### PR TITLE
Add Dockerfile for konflux  adoption

### PR DESCRIPTION
**Pull request by pramodbindal**

### CODE REVIEW

File: openshift/dockerfiles/cache.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+WORKDIR /go/src/github.com/openshift-pipelines/tekton-caches
+COPY . .
+
+RUN go build -v -o /tmp/cache  ./cmd/cache
+
+FROM $RUNTIME
+ARG VERSION=tekton-caches-main
+
+COPY --from=builder /tmp/cache /ko-app/cache
```
Comment (vdemeester) [inline openshift/dockerfiles/cache.Dockerfile:14]: We should do the same we do for things like git-init or entrypoint (`shadow-utils`, …).
- https://github.com/openshift-pipelines/ecosystem-images/blob/main/git-init/Dockerfile#L32
- https://github.com/openshift-pipelines/tektoncd-pipeline/blob/main/openshift/dockerfiles/entrypoint.Dockerfile#L37

Comment (pramodbindal) [reply openshift/dockerfiles/cache.Dockerfile:14]: cache images is executed by script via stepAction.
do we still need an entry point for this ?

Comment (vdemeester) [reply openshift/dockerfiles/cache.Dockerfile:14]: I don't mean the entrypoint, I mean installing `shadow-utils` and doing the addgroup, … that we do in other images.

**Pull request merged**